metasynth/distribution/categorical.py

```diff
@@ -1,11 +1,11 @@
 """Module containing categorical distributions."""
 
-from typing import Sequence, Union
+from typing import Union
 
 import pandas as pd
```

Comment:
dependence on pandas should be removed here (also in line 58 of this file)

Reply:
Are you sure? I think for the annotations, pandas dataframes should be allowed here.